diff --git a/core/lib/Drupal/Core/Form/FormPreprocess.php b/core/lib/Drupal/Core/Form/FormPreprocess.php
index 2a03737bf5..b581917a4e 100644
--- a/core/lib/Drupal/Core/Form/FormPreprocess.php
+++ b/core/lib/Drupal/Core/Form/FormPreprocess.php
@@ -363,6 +363,7 @@ public function preprocessDetails(array &$variables): void {
       $variables['title'] = ['#markup' => $variables['title']];
     }
     $variables['description'] = (!empty($element['#description'])) ? $element['#description'] : '';
+    $variables['description_display'] = $element['#description_display'] ?? 'after';
     $variables['children'] = (isset($element['#children'])) ? $element['#children'] : '';
     $variables['value'] = (isset($element['#value'])) ? $element['#value'] : '';
     $variables['required'] = !empty($element['#required']) ? $element['#required'] : NULL;
